### ISSUE TITLE
Show full commit message in Git history

### ISSUE THREAD

**Issue by kent37** [rstudio/rstudio#6796]
Please show the full commit message (not just the first line) in the details pane (below the list of commits) of the Git History view.

**Comment by NBordag** [comment]
yes please, it would appreciate that feature as well, thank you for oyur hard work
